acceptance/tests/inventory/basic_fact_retrieval.rb

```diff
@@ -64,5 +64,27 @@
       facts = parse_json_with_error(result.stdout)
       assert_equal(structured_data, facts.first["value"])
     end
+
+    step "create a fact with a null byte" do
+      time = Time.now.iso8691
```

Comment:
This function makes it so you don't have to remember the iso number https://github.com/puppetlabs/puppetdb/blob/master/puppet/lib/puppet/util/puppetdb.rb#L34